lib/twitter.js

```diff
@@ -161,7 +161,7 @@ Twitter.prototype.post = function(url, content, content_type, callback) {
     }
     else {
       try {
-        var json = JSON.parse(data || {});
+        var json = data ? JSON.parse(data) : data;
         callback(null, json);
```

Comment:
What would it return in case of `!data`? Wouldn't be better to ensure that always a `null` is returned when there is not data?